src/custom-components/ColorPickerModal.tsx

```diff
@@ -0,0 +1,163 @@
+import { gamepadSliderClasses, ConfirmModal, SliderField } from "../deck-components";
+import { useState, VFC, CSSProperties } from "react";
+
+interface ColorPickerModalProps {
+  closeModal: () => void;
+  onConfirm?(HSLString: string): any;
+  title?: string;
+  defaultH?: number;
+  defaultS?: number;
+  defaultL?: number;
+  defaultA?: number;
+}
+
+export const ColorPickerModal: VFC<ColorPickerModalProps> = ({
```

Comment:
IIRC this should be an FC, not a VFC

Reply:
I have addressed and fixed this in my newest commit to the repo.
To be completely honest, I barely know the difference and use them interchangeably, so thank you for pointing it out.